lang/clisp/Portfile

```diff
@@ -25,7 +25,7 @@ long_description        \
   CLUE/CLIO. Command line editing is provided by readline. CLISP runs \
   Maxima, ACL2 and many other Common Lisp packages.
 
-homepage        https://www.clisp.org
+homepage        http://clisp.org
```

Comment:
This site simply redirects to `https://clisp.sourceforge.io`. So perhaps we should use that instead?

Reply:
+1 — That seems sane.

Reply:
I picked http://clisp.org merely since that is what is shown in bold on the homepage, but I don’t have a preference and will go with https://clisp.sourceforge.io if that is what others and/or the maintainer prefer.

Reply:
> I picked http://clisp.org merely since that is what is shown in bold on the homepage, but I don’t have a preference and will go with https://clisp.sourceforge.io if that is what others and/or the maintainer prefer.

Yep, the latter is less likely to require any lashings with a wet noodle!  ;-)